File: cs/proxy/db/RebateDialOperation.cs
/// <summary>
/// 返利轮盘
/// </summary>
namespace org.zgl.service.client.hall{
	[Proxy(gameId = -7)]
	public interface RebateDialOperation : IHttpSyncService { 
		/// <summary>
		/// 抽奖
		/// </summary>
		RebateDialDto luckyDraw(long arg0);
		/// <summary>
		/// 查看轮盘信息
		/// </summary>
		RebateDialInfoDto dialInfo(long arg0);
	}
}

File: componet/src/main/java/org/zgl/service/client/hall/RebateDialOperation.java
package org.zgl.service.client.hall;

import org.zgl.dto.clinet.db.RebateDialDto;
import org.zgl.dto.clinet.db.RebateDialInfoDto;
import org.zgl.proxy.ClassDesc;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.IHttpSyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/17
 * @文件描述：
 */
@ClassDesc("返利轮盘")
public interface RebateDialOperation extends IHttpSyncService {
    @MethodDesc("抽奖")
    RebateDialDto luckyDraw(long uid);
    @MethodDesc("查看轮盘信息")
    RebateDialInfoDto dialInfo(long uid);
}
